force-app/main/default/lwc/appFieldSet/appFieldSet.js

```diff
@@ -0,0 +1,363 @@
+			}
+		});
+
+		let allData = this.fieldSetData;
+		this.template.querySelectorAll('c-lookup').forEach(function (lookup) {
+			allData.forEach(function (data) {
+				if (data.fieldPath === lookup.customKey) {
+					if (lookup.getSelection().length > 0 && data.fieldPath) {
+						let initialSelection = {
+							id: lookup.getSelection()[0].id,
+							sObjectType: lookup.getSelection()[0].sObjectType,
+							icon: 'standard:account',
+							title: lookup.getSelection()[0].title,
+							subtitle: lookup.getSelection()[0].subtitle
+						};
+						data.lookupConfig.initialSelection = [initialSelection];
+						newObjectToInsert[data.fieldPath] = lookup.getSelection()[0].id;
+					}
+				}
+			});
+		});
+
+		this.fieldSetData = [...allData];
+
+		return newObjectToInsert;
+	}
+
+	validateForm() {
+		const isValid = [...this.template.querySelectorAll('lightning-input')]
+			.reduce((validSoFar, inputFields) => {
+				inputFields.reportValidity();
+				return validSoFar && inputFields.checkValidity();
+			}, true);
+
+		return isValid;
+	}
+
+	//Variants usage - error, success, warning
+	handleToastMessage(title, message, variant) {
+		this.dispatchEvent(
+			new ShowToastEvent({
+				title: title,
+				message: message,
+				variant: variant
+			})
+		);
+	}
+
+	cancel() {
+		this.fieldSetData.forEach(function (field) {
+			field.value = "";
+		});
+
+		this.fieldSetData = [...this.fieldSetData];
+	}
+
+	/* CUSTOM LOOKUP METHOD START  */
+	handleLookupField() {
+		this.fieldSetData.forEach(function (item) {
+			if (item.isLookupField) {
+				item.lookupConfig = {
+					isMultiEntry: false,
+					initialSelection: [],
+					lookupErrors: [],
+					notifyViaAlerts: false // Use alerts instead of toast to notify user
+				};
+			}
+		});
+	}
+
+	handleLookupSearch(event) {
+		let selectedLookupKey = event.currentTarget.customKey;
+		let lookupObject;
+
+		this.fieldSetData.forEach(function (data) {
+			if (data.fieldPath == selectedLookupKey) {
+				lookupObject = data.lookupObject;
+			}
+		});
+
+		let parameters = event.detail;
+		parameters.lookupObject = lookupObject;
+ 		searchGenericLookup(parameters).then((results) => {
+				this.template.querySelectorAll('c-lookup').forEach(function (lookup) {
+					if (selectedLookupKey === lookup.customKey) {
+						lookup.setSearchResults(results);
+					}
+				})
+			}).catch((error) => {
+				this.notifyLookupUser('Lookup Error', 'An error occured while searching with the lookup field.', 'error');
+				// eslint-disable-next-line no-console
+				console.error('Lookup error', JSON.stringify(error));
+				this.lookupConfig.lookupErrors = [error];
+			});
+	}
+
+	handleLookupChange() {
+		this.lookupConfig.lookupErrors = [];
+	}
+
+	validateLookup() {
+		const selection = this.template.querySelector('c-lookup').getSelection();
+		if (selection.length === 0) {
+			this.lookupConfig.lookupErrors = [
+				{ message: 'Please choose your care facility to continue...' }
```

Comment:
Can you change this label?  to please choose your "OBJECT with name (if possible)"   to continue